packages/react-components/review/beta/react-components.api.md

```diff
@@ -234,6 +234,11 @@ export interface CameraSitePermissionsProps extends CommonSitePermissionsProps {
 // @beta
 export type CameraSitePermissionsStrings = SitePermissionsStrings;
 
+// @public
+export type CancelEditCallback = (messageId: string, metadata?: Record<string, string>, options?: {
+    attachedFilesMetadata?: FileMetadata[];
```

Comment:
Curious about the parameters here - if we wanna minimize the callback param, messageId should be enough, contoso can grab the real message from state themselves.

If we want to provide more info and make it easier to be consumed, I guess pass the whole message object would be good as well

if we want to separate properties one by one into the params. IMO, for priority, message content >>  metadata > fileMetadata

Reply:
When I was thinking about this, we have to have the message ID. This would allow a user to look up any other message parts if they need them. 
As there's this signature on the edit, it seemed appropriate to use a similar callback signature here. We don't need the message content however as this is a cancel.

Reply:
Yea, that makes sense to drop those signature - I believe onEditMessage is getting the metadata when someone upload a new file or remove one or two files from the message, so it makes more sense to get them, because they need to be passed to underlying handler and sdk

When I am talking about sdk - it reminds me that all these onEditMessage/onSendMessage is actually coming from underlying handlers and sdk - will we have a new handler for this onCancelMessage  and connect it to sdk? If not, I would like to see what user case we wanna support when expose this api - it seems like not a required api for this error handling feature